packages/@aws-cdk/aws-rds/lib/cluster-engine.ts

```diff
@@ -79,6 +86,14 @@ export interface ClusterEngineFeatures {
    * @default - no s3Export feature name
    */
   readonly s3Export?: string;
+
+  /**
+   * Feature name for the DB instance that the IAM role to invoke a lambda function is to
+   * be associated with.
+   *
+   * @default - no Lambda feature name
+   */
+  readonly lambda?: string;
```

Comment:
I'm a little confused about what this prop is based off the docstring. Could you please clarify?

Reply:
When adding a role to the cluster, there is supposed to be a specific feature name that the role is for. For this functionality, that future name is `Lambda`. 

Refer to step 2 part 4 here: https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/PostgreSQL-Lambda.html#PostgreSQL-Lambda-access. A difficult part is that it is different for PostgreSQL and MySQL.

```
aws rds add-role-to-db-cluster \
       --db-cluster-identifier my-cluster-name \
       --feature-name Lambda \
       --role-arn  arn:aws:iam::444455556666:role/rds-lambda-role   \
       --region aws-region
```